extensions/amp-story/amp-story.md

```diff
@@ -156,6 +193,22 @@ The `amp-story` component represents an entire story.  The component itself  imp
 
 Identifies that the AMP document is a story.
 
+##### title [required]
+
+The title of the story.
+
+##### publisher [required]
+
+The name of the story's publisher.
+
+##### publisher-logo-src [required]
+
+A publisher's logo in square format (1x1 aspect ratio).
```

Comment:
Suggest adding, "For example `publisher-logo-src="https://example.com/logo/1x1.png"`, where `1x1.png` is a 36x36 pixel logo.

Reply:
Done.